src/utils/pattern.ts

```diff
@@ -19,7 +19,7 @@ export function isStaticPattern(pattern: Pattern): boolean {
  * Return true for pattern that looks like glob.
  */
 export function isDynamicPattern(pattern: Pattern): boolean {
-	return isGlob(pattern);
+	return isGlob(pattern, { strict: false });
```

Comment:
Good catch! Thanks 🌮 

Can you write a test for the required behavior with `?` symbol (`isDynamicPattern`)?

https://github.com/mrmlnc/fast-glob/blob/2a3c7985bc1df494fede60cfbcd2dc9574fbc76c/src/utils/pattern.spec.ts#L21-L33

Reply:
@mrmlnc sure, test added